test/declaration_test.dart

```diff
@@ -1374,6 +1426,7 @@ void main() {
   test('Supports', testSupports);
   test('Viewport', testViewport);
   test('Font-Face', testFontFace);
+  test('font-family', testFontFamily);
```

Comment:
I know this doesn't follow the existing pattern, but it would be better practice to make this a `group('font-family', `, and have `testFontFamily` declare a `test` for each test-case in it.

Reply:
Sounds good - done.